addons/l10n_in_sale_stock/models/sale_order.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models, fields, api
+from odoo import models, api
```

Comment:
remove this diff to remove `addons/l10n_in_sale_stock/models/sale_order.py` from the modified files